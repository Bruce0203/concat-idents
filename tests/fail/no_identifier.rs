concat_idents::concat_idents!(fn_name =  {
    fn fn_name() {
        /* idents cannot be nothing */
    }
});

fn main() {}
