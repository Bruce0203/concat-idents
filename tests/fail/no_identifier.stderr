error: Expected at least one identifier
 --> $DIR/no_identifier.rs:1:42
  |
1 | concat_idents::concat_idents!(fn_name =  {
  |                                          ^
